### DISCUSSION TITLE
Best process for importing models and textures into Webots

### DISCUSSION THREAD

**Discussion by GearsAD** [cyberbotics/webots#5525] [Q&A]
We're starting a project using Webots and we're trying to find the best way to import models with UVs and textures from Blender. It seems the Webots exporter is out of date, can you please recommend the best way to do this?

Thanks!
 
Cross-reference: https://github.com/cyberbotics/blender-webots-exporter/issues/12

**Comment by omichel** [comment]
The recommended way to import 3D models, including UV, is to export the Blender model in OBJ or Collada format and use it in Webots with the [CadShape](https://cyberbotics.com/doc/reference/cadshape) node that references the OBJ or Collada file.

**Reply by GearsAD** [reply]
Hi @omichel thanks, we'll try that, appreciate the quick response!
